Cargo.toml

```diff
@@ -17,10 +17,10 @@ default = ["rustls"]
 openssl = ["actix-web/openssl", "awc/openssl"]
 rustls = ["actix-web/rustls", "awc/rustls"]
 shuttle = [
-  "dep:shuttle-actix-web",
-  "dep:shuttle-runtime",
-  "dep:shuttle-static-folder",
-  "dep:tokio",
+  "dep:shuttle-actix-web", 
+  "dep:shuttle-runtime", 
+  "dep:shuttle-static-folder", 
```

Comment:
why the space after the comma?

Reply:
yeah, please revert the changes in `Cargo.*`

Reply:
I was developing on windows on top of git bash and it seemed to do weird things with the formatting with my vscode settings. Sorry about that. I think the package version changes below are still needed though.